events/testdata/kinesis-firehose-response.json

```diff
@@ -5,15 +5,25 @@
        "recordId": "record1",
        "result": "TRANSFORMED_STATE_OK",
        "metadata": {
-         "partitionKeys": {}
+         "partitionKeys": {},
+         "otfMetadata": {
```

Comment:
How does Firehose handle metadata with empty strings? Will this cause an error? IE, if I want to transform but *not* route, what happens?

Reply:
Without knowing Firehose's behavior, it'd be prudent to add 'omitempty' to the new struct tag